Add templates to repo (#282)
mongodb/terraform-provider-mongodbatlas: .github/ISSUE_TEMPLATE/Bug_Report.md

---
name: Bug report
about: Something unexpected happen? Report an error, crash, or an incorrect behavior here.

---

<!--
Hello!

> Thank you for opening an issue.  Please note that we try to keep the Terraform MongoDB Atlas Provider issue tracker reserved for bug reports.  Please ensure you check open and closed issues first to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, the report).  

Reviewer: ```suggestion
Thank you for opening an issue.  Please note that we try to keep the Terraform MongoDB Atlas Provider issue tracker reserved for bug reports.  Please ensure you check open and closed issues first to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, to the issue).  
```

> If your issue relates to Terraform itself, please open it in the Terraform repository https://github.com/terraform-providers.

Reviewer: [q] this link takes me to a profile page and not to a particular repo, I don't think this is very helpful, should this link be https://github.com/hashicorp/terraform/issues ?

Author: Sloppy copy and paste. Will correct!

-->

> ### Terraform CLI and Terraform MongoDB Atlas Provider Version

Reviewer: [q] not related to this but do you think we should ask for OS version/arch as well?

Author: I've never had it be an item we needed so I think we are okay to leave that off - I've also not seen many providers do so.

<!---
Run `terraform version` to show the version, and paste the result for Terraform and the MongoDB Atlas Provider between the ``` marks below.

> If you are not running the latest version of Terraform and the MongoDB Atlas Provider, please try upgrading because your issue may have already been fixed.

Reviewer: [nit] this is a bit condescending 
```suggestion
If you are not running the latest version of Terraform and the MongoDB Atlas Provider, please try to reproduce the issue in a more recent version in case it has been fixed already.
```

-->

```hcl
# Copy-paste your version info here
```

### Terraform Configuration File 
<!--
Paste the relevant parts of your Terraform MongoDB Atlas Provider configuration file info between the ``` marks below.

> Please be sure to redact any sensitive information.  If longer consider creating a GitHub Gist for the information.

Reviewer: ```suggestion
Paste the relevant parts of your Terraform MongoDB Atlas Provider configuration file info between the ``` marks below or consider creating a GitHub Gist with the information and share it here.
Please be sure to redact any sensitive information like API Keys or secrets.
```

Author: I bounced around on how to phrase this.  Going to use this as a guide and adjust a bit more.

-->

```hcl
# Copy-paste your configuration info here
```

### Debug Output
<!--
Full debug output can be obtained by running Terraform with the environment variable `TF_LOG=trace`. Please create either a GitHub Gist or attach a file containing the debug output. Please do _not_ paste the debug output in the issue, since debug output is long.

Debug output may contain sensitive information. Please review it before posting publicly, and if you are concerned feel free to redact it.
-->

### Crash Output
<!--
If the console output indicates that Terraform crashed, please either share a link to a GitHub Gist containing the output of the `crash.log` file or attach the file.
-->

### Expected Behavior
<!--
What should have happened?
-->

### Actual Behavior
<!--
What actually happened?
-->

### Steps to Reproduce
<!--
Please list the full steps required to reproduce the issue, for example:
1. `terraform init`
2. `terraform apply`

> -->

Reviewer: I would probably move these before asking for logs and crash reports, I'm not sure with terraform but some issues may not require crash/debug logs and just steps to repro may be enough, and asking those upfront may be more challenging for people who may have a clear idea of the bug/steps to repro

Author: This is another area where that seems to be the common flow but what you describe is probably way more common.
